react/components/Image.js

```diff
@@ -4,18 +4,46 @@ import PropTypes from 'prop-types'
 import productSummary from '../productSummary.css'
 
 /** Image component with 1:1 aspect ratio */
-export default function Image({ src, className }) {
+export default function Image({ src, className, width, height }) {
+  const resizeSrc = src => {
```

Comment:
Couldn't we reuse [this](https://github.com/vtex-apps/product-details/blob/master/react/utils/generateUrl.js)?

Reply:
We could, but how?

Reply:
You could:

1) import the function exported from `product-details`'s utils.
2) create a common component on `store-components` with the algorithm.

I will check with @brunojdo

Reply:
I think that inserting a dependency of `product-details` only for that is too costly

Reply:
This same function is being used on **minicart** too.
https://github.com/vtex-apps/minicart/blob/master/react/utils/urlHelpers.js
This function should be a lib or something that allow using the same code.